asset.go

```diff
@@ -311,6 +311,9 @@ type External struct {
 		// Coingecko's asset ID, if relevant
 		AssetId string `yaml:"asset_id,omitempty"`
 	} `yaml:"coin_gecko,omitempty"`
+	IndexingCo struct {
+		ChainId string `yaml:"chain_id,omitempty"`
```

Comment:
leave a comment where to find so we don't forget

```go
// IndexingCo ID's can be found at https://jiti.indexing.co/networks
```

Reply:
Fixed, thanks!